api/pkg/transformer/feast/validator_test.go

```diff
@@ -753,7 +753,7 @@ func TestValidateTransformerConfig(t *testing.T) {
 					},
 				},
 			},
-			wantError: errors.New("udf compilation failed: unknown func unknown (1:1)\n | unknown()\n | ^"),
+			wantError: errors.New("udf compilation failed: unknown name unknown (1:1)\n | unknown()\n | ^"),
```

Comment:
Is this intended change?

Reply:
yes, this is due to update on `expr` library